File: 02_standard_deviation/task/code.js
var x = [];
var sd = (n = ave = a = b = i = 0);
// i = 0
// n = input("enter nums of nums do you have")
// a = 0
// b = 0
// ave = a/n
// = 0
// total = 0

n = parseInt(prompt("How many Items?"));

for (var i = 0; i < n; i++) {
  x[i] = +prompt("Give me items ?");
  ave += x[i];
}
ave = ave / n;
console.log(ave);

for (let j = 0; j < n; j++) {
  a += Math.pow(x[j] - ave, 2);
  console.log(a);
}

b = a / n;

sd = Math.sqrt(b);
console.log(sd);

// var zz = ["A", "B", "C"];

// console.log(zz[0]);
// console.log(zz[1]);
// console.log(zz[2]);

File: 02_standard_deviation/task/code.py
import math

x = []
sd = n = ave = a = b = i = 0


n = int(input("How many Items?"))

for i in range(n):
    x.append(float(input("Give me item: ")))
    ave += x[i]

ave = ave / n

print("Average:", ave)

for j in range(n):
    a += (x[j] - ave) ** 2
    print(a)

b = a / n
sd = math.sqrt(b)

print("Standard Deviation", sd)